Makefile

```diff
@@ -63,10 +62,6 @@ test: $(NODEUNIT)
 	$(NODEUNIT) test/*.test.js
 	$(MOCHA) test/plugins/*.test.js
 
-.PHONY: nsp
```

Comment:
I think we should do `npm audit` instead of removing this check altogether.

Reply:
`npm audit` runs on every `npm install`.